src/views/Graph.vue

```diff
@@ -86,6 +86,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             />
           </g>
         </g>
+        <g v-if="groupCyclePoint" class="graph-subgraph-container">
+          <GraphSubgraph
+            v-for="(subgraph, index) in subgraphs"
+            :key="index"
```

Comment:
It's not an array, it's an object
```suggestion
            v-for="(subgraph, key) in subgraphs"
            :key="key"
```